### PR TITLE
Addresses Style Guide Issue #173

### PR DESCRIPTION
**Pull request by nmuller66**
Add information on verbiage for recommendations to address Issue #173.

### CODE REVIEW

File: en-US/Design.xml [hunk 1]
```diff
@@ -721,6 +721,34 @@ $ vi myFile.txt
 		</para>
 
 	</section>
+
+	<section id="making-recommendations">
+	 <title>Making Recommendations</title>
+		<para>
+		When making a recommendation, the preferred verbiage is <emphasis role="bold">"Red&nbsp;Hat recommends..."</emphasis> instead of the common but indirect "It is recommended...".
```
Comment (daobrien) [inline en-US/Design.xml:728]: Do we need `<emphasis role="bold">`?

File: en-US/Design.xml [hunk 2]
```diff
@@ -721,6 +721,34 @@ $ vi myFile.txt
...
+		Recommendations can include best practices, recommended practices, and product-specific suggestions.
+		Refer to <xref linkend="avoiding-confusing-language"/> for information on using the terms "best practices" and "recommended practices" in Red&nbsp;Hat documentation.
+	  </para>
+    <para>
+	   <emphasis role="strong">Example 1</emphasis>
```
Comment (daobrien) [inline en-US/Design.xml:733]: `<emphasis role="strong">` ?
You might be able to make use of `<example>` elements here. Have a look at how IBM does their correct/incorrect examples; maybe we can borrow their idea.

File: en-US/Design.xml [hunk 3]
```diff
@@ -721,6 +721,34 @@ $ vi myFile.txt
...
+	  </para>
+		<para>
+			Incorrect: "Although the attack surface is reduced to the same-project traffic, it is recommended to create multiple service accounts within a project."
+		</para>
+		<para>
+			Correct: "Although the attack surface is reduced to the same-project traffic, Red&nbsp;Hat recommends creating multiple service accounts within a project."
+		</para>
+	 	<para>
+		 <emphasis role="strong">Example 2</emphasis>
```
Comment (daobrien) [inline en-US/Design.xml:742]: Same comment as before. Also remember to use `&nbsp;` with "Red Hat."

File: en-US/Slang.xml [hunk 4]
```diff
@@ -23,8 +23,10 @@
 					<para>
 						This is a commonly understood phrase, and despite some concerns about using superlatives without statistics to back them up, Red&nbsp;Hat does not actively discourage its use. It is also a more common search term than most alternatives. If you are in any doubt, the preferred alternative is "recommended practices."
 					</para>
-
-				</listitem>
+					<para>
+						See the section <xref linkend="making-recommendations" /> for additional information about recommendations in Red&nbsp;Hat documentation.
```
Comment (daobrien) [inline en-US/Slang.xml:27]: This actually renders as "See the section Section 3.10, “Making Recommendations” for additional information" so you can remove "the section" from your text.

File: en-US/Design.xml (hunk 1 above)
Comment (nmuller66) [reply en-US/Design.xml]: Fixed.

File: en-US/Design.xml (hunk 2 above)
Comment (nmuller66) [reply en-US/Design.xml]: Updated to use <example>

File: en-US/Design.xml (hunk 3 above)
Comment (nmuller66) [reply en-US/Design.xml]: Fixed

File: en-US/Slang.xml (hunk 4 above)
Comment (nmuller66) [reply en-US/Slang.xml:27]: Fixed.

**Comment by nmuller66** [comment]
Requested updates complete.

**Pull request merged**